call-a-function-api-oci-clouddev.md

```diff
@@ -0,0 +1,772 @@
+* Create an API Gateway for your function
+* Call your function from the internet using your API Gateway.
+![] {% img [classes] assets/call-a-func-oracle-funcs-api-gtw-diagram.png [1200 675] "" "Oracle Cloud Infrastructure Diagram showing a function executing on the service, the API gateway getting the request and returning the function, with registry in tenancy and VCN within the tenancy allowing access to the Internet" %}(https://docs.oracle.com/en-us/iaas/developer-tutorials/tutorials/functions/func-api-gtw/images/)
+
+For additional information, see:
+
+* [Start for Free]({{ site.urls.always_free }})
+* [Command Line Interface (CLI)](https://docs.oracle.com/iaas/Content/API/Concepts/cliconcepts.htm)
+* [Oracle Functions](https://docs.oracle.com/iaas/Content/Functions/Concepts/functionsoverview.htm)
+* [Oracle API Gateway](https://docs.oracle.com/iaas/Content/APIGateway/Concepts/apigatewayoverview.htm)
+
+## Before You Begin
+
+To successfully perform this tutorial, you must have the following:
+
+### OCI Account Requirements
+
+* A **paid** Oracle Cloud Infrastructure account. See [Signing Up for Oracle Cloud Infrastructure](https://docs.oracle.com/iaas/Content/GSG/Tasks/signingup.htm). 
+* Your OCI account configured to support Oracle Functions development. See [Oracle Functions on Cloud Shell Quickstart](https://www.oracle.com/webfolder/technetwork/tutorials/infographics/oci_functions_cloudshell_quickview/functions_quickview_top/functions_quickview/index.html#).
+* Completion of one of the two Oracle Functions introduction tutorials. 
+    * [Functions: Get Started using Cloud Shell](https://docs.oracle.com/iaas/developer-tutorials/tutorials/functions/func-setup-cs/01-summary.htm#setup-functions-dev-cs)
+    * [Functions: Get Started using the CLI](https://docs.oracle.com/iaas/developer-tutorials/tutorials/functions/func-setup-cli/01-summary.htm#setup-functions-dev)
+* Completing one of the two tutorials results in: 
+    * Oracle Functions is set up and configured to create applications and deploy functions.
+    * Oracle Registry is set up to store function images.
+    * Docker is logged into the Oracle Registry.
+    * The required VCN and required resources needed for Oracle Functions.
+    * An API key pair and an auth token.
+
+### Software Requirements
+
+**Oracle CLI**
+
+* Python 3.6+ and pip3.
+* Docker Engine: A Linux computer or Linux VM. See [Docker engine requirements](https://docs.docker.com/engine/install/) for versions, and distros supported.
+* Docker Desktop: Available for MacOS or Windows 10. 
+    * Windows 10: Windows 10 update 2004 with WSL 2 and Ubuntu or other distro installed. 
+    * See [Windows Subsystem for Linux Installation Guide for Windows 10](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+    * [Install Docker Desktop for Windows 10.](https://docs.docker.com/docker-for-windows/install/)
+
+    > **Note:** Docker includes special Linux support for WSL 2 on Windows 10 update 2004.
```

Comment:
This blockquote needs to be indented 4 spaces to become part of its parent list item (just like a paragraph would). The `{:.notice}` liquid tag should be indented to match the start point of the `>` on the line above.